public/scripts/landing.js

```diff
@@ -4,7 +4,35 @@ class Landing extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = { gameId: '' }
+    this.state = { gameId: '',
+                   proposedID: '',
+                   invalidID: ''}
+    
+    txtFile = "../../wordlist.txt"          //maybe filepath needs fixing
+    file = new File(txtFile)
+    file.open("r") // open file with read access
+    wordArray = []
+    i = 0
```

Comment:
`i` never increases?
also consider using the standard stack overflow solution to read a fie's contents and write into an array